test/Microsoft.ML.TestFramework/DataPipe/TestDataPipe.cs

```diff
@@ -103,7 +103,7 @@ public void SavePipeLabelParsers()
             TestCore(pathData, true,
                 new[] {
                     "loader=Text{col=RawLabel:TXT:0 col=Names:TXT:1-2 col=Features:TXT:3-4 header+}",
-                    string.Format("xf=TermLookup{{key=- col=FileLabel:RawLabel data={{{0}}}}}", mappingPathData),
+                    string.Format("xf=TermLookup{{valuesAsKeyType=- col=FileLabel:RawLabel data={{{0}}}}}", mappingPathData),
```

Comment:
>valuesAsKeyType [](start = 50, length = 15)

I left comment in ValueMapper regarding Argument class,please revert this change as well.